src/Tools/AnalyzerRunner/Program.cs

```diff
@@ -45,7 +45,21 @@ public static async Task Main(string[] args)
                     cts.Cancel();
                 };
 
+#if NETCOREAPP
+            // Since we are running as a dotnet tool we should be able to find an instance of
+            // MSBuild in a .NET Core SDK.
+            var msBuildInstance = MSBuildLocator.QueryVisualStudioInstances().First();
```

Comment:
FYI this is conflicting with your other PR.

Reply:
Hence why I've marked this pull request as blocked 😄